### PR TITLE
refactor(DB): Unify two player stat tables.

### PR DESCRIPTION
**Pull request by heyitsbench**
## Changes Proposed:
This PR proposes changes to:
-  [X] Database (SAI, creatures, etc).

This PR corrects a case of laziness with 5534455c3248ea88103abe1acb15d0a42ff9ae48 and unifies two tables that have no need being split.

## Issues Addressed:

- Closes unreported issue.

## SOURCE:

The changes have been validated through:
- [X] Source is not applicable to this change.

## Tests Performed:

This PR:
- [X] Has been tested in-game by the author.
- [X] May have edge cases to be tested.

## How to Test the Changes:

- [X] This pull request may require further testing.

1. Make characters.
2. Play around with various class/level combinations.
3. Play around with gear combinations.
4. Observe health/mana values.

## Known Issues and TODO List:

- [X] ~~It may be preferable to combine the two functions that load from this table into one, as there is not much benefit to keeping them separate. That is outside the scope of this PR.~~ These functions have now been condensed.
- [ ] The stats being moved around may need corrections similar to the original commit. That is outside the scope of this PR.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Kitzunu** [comment]
Wow bench doing PRs pog

**Comment by heyitsbench** [comment]
> Wow bench doing PRs pog

Bench "fixing" his past PRs pog.

File: src/server/game/Globals/ObjectMgr.cpp [hunk 1]
```diff
@@ -4067,12 +4067,12 @@ void ObjectMgr::LoadPlayerInfo()
     {
         uint32 oldMSTime = getMSTime();
 
-        //                                                0      1      2       3
-        QueryResult result  = WorldDatabase.Query("SELECT class, level, basehp, basemana FROM player_classlevelstats");
+        //                                                  0      1      2        3
+        QueryResult result  = WorldDatabase.Query("SELECT Class, Level, BaseHP, BaseMana FROM player_class_stats");
```
Comment (Kitzunu) [inline src/server/game/Globals/ObjectMgr.cpp:4071]: Okay, but why not initialize this at L#4186 with the rest of the player_class_stats stuff?

Comment (Kitzunu) [reply src/server/game/Globals/ObjectMgr.cpp:4071]: Don't be lazy, let's not query the same table twice when we don't need to :D

**Review by Kitzunu — APPROVED** [review]

**Comment by Kitzunu** [comment]
@azerothcore/testers can someone take a quick look at this?

**Pull request merged**